src/product.js

```diff
@@ -0,0 +1,31 @@
+class Product {
+  id = 0;
+  name = "";
+  products = [];
+}
+
+const ioc = Object.freeze({
+  product: new Product(),
+});
+
+class ProductService {
+  products = []

```

Comment:
what you do mean by? **Could you separate entities from service and ioc implementation?**

You mean remove the product object from inside the service and ioc below when using the service?, can you clarify this for me please

Reply:
Yeah, so create ioc on its own file and create create a pr only for this. You can remove product service for now, we don't need it 9lin this moment